packages/react-components/react-popover/src/components/Popover/Popover.types.ts

```diff
@@ -115,8 +115,16 @@ export type PopoverProps = Pick<PortalProps, 'mountNode'> & {
    * where the focus trap involves setting outside elements inert.
    *
    * @default false
+   * @deprecated this behavior is default provided now, to opt-out of it in favor of standard behavior use the `inertTrapFocus` property
```

Comment:
should the default be `true`? 🤔 I'm not actually sure since we've just made this be default behaviour

Reply:
I'm not even using the property internally anymore, so perhaps it shouldn't even have a `default` in the documentation?!

Reply:
sounds good, let's remove default documentation